versatileimagefield/templates/django/forms/widgets/attrs.html

```diff
@@ -0,0 +1 @@
+{% for name, value in widget.attrs.items %}{% if value != False %} {{ name }}{% if value != True %}="{{ value }}"{% endif %}{% endif %}{% endfor %}
```

Comment:
The changes to the `if` template tag adding support for `is` and `is not` are in 1.9 I think, so this doesn't work on 1.8 -- I added this change to continue supporting 1.8.